apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/style-tour.scss

```diff
@@ -154,6 +155,13 @@ $welcome-tour-button-background-color: #32373c; // former $dark-gray-700. TODO:
 	}
 }
 
+@include break-mobile {
```

Comment:
The media query here applies to anything wider than 480px, so may not be what we want?